pkg/server/container_status.go

```diff
@@ -70,6 +74,20 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 			reason = errorExitReason
 		}
 	}
+	if verbose {
+		info = make(map[string]string)
+		m, _ := json.Marshal(status.Pid)
+		info["pid"] = string(m)
+		m, _ = json.Marshal(meta.Config)
+		info["container config"] = string(m)
```

Comment:
config.

Reply:
feel it's nice to explain in more detail since it's human readable stuff